theme/src/css/_zendesk-elements.scss

```diff
@@ -143,6 +143,7 @@ textarea,
 
 .label-sm {
 	@include clay-label-size($label-sm);
```

Comment:
Add a line break between mixin and CSS property.  See https://github.com/rdai10/zendesk/blob/123c9b5f26613df574cd25ba2edeaf4e5a07e7cd/theme/src/css/_zendesk-notification.scss#L2...L4 for an example.

Reply:
Will do!

Reply:
https://github.com/rdai10/zendesk/pull/71

Reply:
Thanks @rdai10!